### ISSUE TITLE
[BUG] AttributeError: module 'FastGeodis' has no attribute '__version__'

### ISSUE THREAD

**Issue by londumas** [masadcv/FastGeodis#56]
When running the following standard python code:
```python
import FastGeodis
print(FastGeodis.__version__)
```

We receive the following error

```bash
AttributeError: module 'FastGeodis' has no attribute '__version__'
```

`__version__` is a standard and usefull attribute in Python

**Comment by masadcv** [comment]
Thanks for reporting this. I will look into this.

**Comment by masadcv** [comment]
I have added capability for versioning to master branch. This will be released soon as version 1.0.5
Note that due to the pypi delivery, you will only be able to query `__version__` string from version 1.0.5 onwards.
